esp-wifi/src/common_adapter/common_adapter_esp32s2.rs

```diff
@@ -59,7 +60,7 @@ pub(crate) fn phy_mem_init() {
 }
 
 pub(crate) unsafe fn phy_enable() {
-    let count = PHY_ACCESS_REF.fetch_add(1, atomic_polyfill::Ordering::SeqCst);
+    let count = PHY_ACCESS_REF.fetch_add(1, Ordering::SeqCst);
```

Comment:
These `fetch_X` calls will use cas under the hood. @bjoernQ do you know how often `phy_enable` and friends are called?

Reply:
once :)

Reply:
I did miss this, though, so I might want to go through this again.